internal/envtest/webhooks.go

```diff
@@ -74,11 +74,25 @@ func initWebhookInstallOptions() envtest.WebhookInstallOptions {
 	if err != nil {
 		klog.Fatalf("Failed to append controlplane controller webhook config: %v", err)
 	}
+
+	var localServingHostExternalName string
```

Comment:
I would like to make this configurable via env var (and add a script in hack/* which helps to setup the env var correctly on Mac/LInux